### DISCUSSION TITLE
Eurorack build format

### DISCUSSION THREAD

**Discussion by Lonolo** [probonopd/MiniDexed#594] [Show and tell]
MiniDexed in Eurorack format, 8HP using the elusive Zero2. 
Getting 8 instances per module. 
The depth is less than 30mm.

![4-racke-dXeus-machinas](https://github.com/probonopd/MiniDexed/assets/1200412/ac897611-eed1-4091-b210-62d486f8d82d)
![dXeus-Front](https://github.com/probonopd/MiniDexed/assets/1200412/ae2eac49-8860-4f72-8aab-088013cdd138)
![dXeus-Side-01](https://github.com/probonopd/MiniDexed/assets/1200412/77fbef2c-99a6-40ae-b106-6a42b726f82e)
![dXeus-Back](https://github.com/probonopd/MiniDexed/assets/1200412/f87ddef0-05ff-4a89-8e2a-7a6278c052ab)
![dXeus-Side-02](https://github.com/probonopd/MiniDexed/assets/1200412/78d62c8d-afba-420c-9982-f77c5e30644f)

**Comment by diyelectromusic** [comment]
Love it!  Looks great :)

Are your MIDI sockets connected via headers?  If so could you do an "internal wiring THRU" as an option using jumper wires?

Would love to see a photo of your custom PCB :)

Kevin

**Reply by Lonolo** [reply]
Thanks, been iterating over features and options, and then finally said go minimal and just do it.
 
Yes, those headers are a 3x1 for Thru and 2x1 for In. Those wires could be soldered to the circuit board, the headers make it easier to assemble/reassemble if needed. Was hard to photo in focus, but the pin headers are labeled "02  04  05  I4  I5" and the pin #'s around the In connector for reference when putting it together.

No jumpers on this, but Thru is configured in the minidexed.ini file as: 
  
MIDIThru=ttyS1,ttyS1 

…if it were changed to:

MIDIThru=ttyS1,umidi1

Thru would be routed to the USB port. I think it would be ok to leave it connected,  the Thru on the panel would just not send anything. Is that what you meant by the other option to do an "internal wiring thru”?

![MIDI-HEADERS](https://github.com/probonopd/MiniDexed/assets/1200412/7e17e8c2-8f26-4dcb-8e95-46ba392725ca)

**Reply by diyelectromusic** [reply]
I was assuming your MIDI links went from the THRU from one module to the IN of the next?  If so, I was wondering if those links could simply be made via standard jumper wires inside rather than external MIDI cables to keep things internal, and it looks like it can as long as external leads aren't used.

Anyway, a great build as I say.  Well done - looks fab.

Kevin

**Reply by Lonolo** [reply]
Thanks Kevin,

I follow what you're saying. That's a great idea, cleaner and save $ on midi cables. If I had realized that was an option for connecting multiple modules I would have labeled that for clarity to account for 'human error' connecting them. So yes you can still do this but with caution, perhaps using a two colored jumper for visual verification: 
Master module MIDI OUT pin 5 -> Slave MIDI IN pin5
Master module MIDI OUT pin 4 -> Slave MIDI IN pin4

/Ron

![dXeus-MIDI-Linked](https://github.com/probonopd/MiniDexed/assets/1200412/5b8169c8-31e9-4220-ac46-4ef2bfc7034d)

**Comment by diyelectromusic** [comment]
I'm now thinking how mad it would be to stack two Pi Zero 2s back to back and use something like my proposed MiniDexed extender extension (https://github.com/probonopd/MiniDexed/discussions/504) to give 16 TGs on each Eurorack module!  It would make them a bit deeper though... :)

And if would really set it off with a TX816 vibe if there was a red OLED display (but I don't know if you can get a ssd1306 display in red!)

Kevin

**Reply by Lonolo** [reply]
I have been looking for those SSDs in red they don't exist. But I've considered using a lighting gel although that would make it just look cheap.

Going to check out your extender ...
